### PR TITLE
Run express app in aws

### PR DESCRIPTION
**Pull request by ThomasAndrewMacLean**
### Running in AWS

I tried using the suggested serverless-express module, and this did set up the lambda functions and api-gateway but did not function. Every request would result in a Timeout. I followed their readme with a sample hello-world project and got the same problem. I will upload this demo and file an Issue on their Github page.

As an alternative I set this project up with claudia.js, a package I have used before.

#### Prerequisites
Before you start you need to create an AWS profile and save it's credentials on your computer [more info here](https://claudiajs.com/tutorials/installing.html)

#### First run: setting everything up

The first time you set this up you have to run the command ```npm run createLambda```
This will set up the API gateway and create the lambda function. After it has run you will find the URL in your terminal.

#### Update your application

After this, after every change you make to your Express-app you can run the command ```npm run updateLambda``` and it will update the routes and functionality.

[documentation claudia-express](https://claudiajs.com/tutorials/serverless-express.html)

### Testing on my AWS

[This is my URL, that I will keep up for 1 week](https://0igxzicvej.execute-api.eu-west-1.amazonaws.com/latest)

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -30,5 +30,6 @@ app.post('/zip', (req, res) => {
     }
   })
 })
-app.listen(port, () => console.log(`Service listening on port ${port}!`))
+//We don't listen anymore, just export it.
+//app.listen(port, () => console.log(`Service listening on port ${port}!`))
```
Comment (vkhazin) [inline app.js]: That would break the existing functionality running in a Docker container.

**Review by vkhazin — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (ThomasAndrewMacLean) [reply app.js]: Updated the code, so it also still listens to the port

Comment (vkhazin) [reply app.js]: Cool, I will test and let you know, thanks!

Comment (ThomasAndrewMacLean) [reply app.js]: I don't know how big of a task you are trying to run on the post /zip route. But keep in mind lambda functions have got a timeout (standard this is set to 3 seconds)

File: claudia.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "lambda": {
+    "role": "sfx-creator-service-executor",
```
[the remark this answers is not in the document]
Comment (ThomasAndrewMacLean) [reply claudia.json]: under Detailed info about credentials [here](https://claudiajs.com/tutorials/installing.html) you can see about the user profile. This user profile creates the IAM role/API gateway/lambda. This role is used to write logs

**Pull request merged**